### PR TITLE
feat: update readme about kafka broker option

### PR DESCRIPTION
**Pull request by DaraDoyoung**
- added additional config option to use Kafka feature

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,34 +1,51 @@
-# Aptos Indexer Client Guide
+# Aptos Indexer Processors (Fork)
+This repository is a fork of the [Aptos Indexer Processors](https://github.com/aptos-labs/aptos-indexer-processors) project, aimed at enhancing the existing functionalities and improving performance. The project is designed to efficiently index and process data from the Aptos blockchain.
 
-This guide will get you started with creating an Aptos indexer with custom parsing. We have several endpoints that provided a streaming RPC of transaction data.
+## Enhanced Features
+- **Fungible Asset Balances Performance Improvement**: 
+The performance of the `fungible_asset_balances` table has been optimized. This enhancement allows for faster queries and improved overall efficiency, and the updated table is now exposed through Hasura.
```
Comment (jimmyc256) [inline README.md]: Numerous optimizations has been made to existing postgres tables to increase GraphQL query performance, and as a result, 'fungible_asset_balances' table is now exposed in GraphQL schema.

Comment (DaraDoyoung) [reply README.md]: 제목으로 
Fungible Asset Balances Performance Improvement  -> Postgres Performance Improvement
이 좀 더 좋을까요?

File: README.md [hunk 2]
```diff
@@ -1,34 +1,55 @@
...
+    - testnet: https://grpc.testnet.aptoslabs.com:443
   - `grpc_data_stream_api_key`: Replace `YOUR_TOKEN` with your auth token.
   - `db_connection_uri`: The DB connection used to write the processed data
   - (optional) `starting-version`
     - If `starting-version` is set, the processor will begin indexing from transaction version = `starting_version`.
     - To auto restart the client in case of an error, you can cache the latest processed transaction version. In the example, the processor restarts from cached transaction version that is stored in a table, and if neither `starting_version` nor cached version are set, the processor defaults starting version to 0.
+  - (optional) `brokers`
+    - endpoint of the Kafka brokers
+    - If brokers are set, the processor will publish the processed data to the Kafka topic.
+    - Name of Kafka topic is `aptos.{mainnet/testnet}.{processor name}`.
```
Comment (bbaktaeho) [inline README.md]: @DaraDoyoung  How about changing `processor name` to `model name`?

File: README.md (hunk 1 above)
Comment (jimmyc256) [reply README.md]: 좋습니다.

**Review by jimmyc256 — APPROVED** [review]
LGTM

**Pull request merged**